plugins/topology/README.md

```diff
@@ -145,6 +145,23 @@ The Topology plugin enables you to visualize the workloads such as Deployment, J
     app.openshift.io/vcs-ref: <GIT_REPO_BRANCH>
   ```
 
+  > Note: If Red Hat OpenShift Dev Spaces is [installed and configured](https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/3.7/html/administration_guide/installing-devspaces) and Git URL annotations are also present on the workload YAML file, then clicking on the edit code decorator redirects you to the Red Hat OpenShift Dev Spaces instance.
+
+  > Note: When you deploy the application using the OCP Git import flows, then you do not need to add the labels as import flows add the labels to the workload YAML file. Otherwise, you need to add the labels manually to the workload YAML file.
```

Comment:
```suggestion
When you deploy the Backstage application using the OCP git import flows, you do not need to add the labels as import flows. Otherwise, you need to add the labels to the workload YAML file manually.
```

Reply:
Instead of `Backstage` application I have used `your`. LMK what you think ?

Reply:
I'd suggest explicitly stating the name of the application for the user